roles/nodepool/files/etc/nodepool/elements/bonnyci-nodepool/post-install.d/60-install-nodepool-scripts

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# dib-lint: disable=setu setpipefail
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -e
+
+# always make the nodepool-scripts directory even if nothing goes in it
+mkdir /opt/nodepool-scripts
```

Comment:
this will fail if the folder already exists. Is there any chance of that happening?